File: challengeOne.js
function fizzBuzz(n){
    let random = []
    console.log('wtf')
    for(let i = 1; i < n; i++){
        var temp = '';
        if(i % 3 === 0){
            temp += 'fizz';
        }
        if(i % 5 === 0){
            temp += 'buzz';
        }
        if(temp === ''){
            random.push(i);
        }
        else {
            random.push(temp);
        }
    }
    return random;
}
var test = fizzBuzz(100);
console.log(test);

File: challengeOne.py
def fizzBuzz(n):
    result = []
    for i in range(1, n + 1):
        add = ''
        if i % 3 == 0:
            add += 'fizz'
        
        if i % 5 == 0:
            add += 'buzz'
        
        if add == '':
            result.append(i)
        else:
            result.append(add);
    return result

text = fizzBuzz(20)
print(text)
